CHANGELOG.md

```diff
@@ -3,6 +3,27 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [2.1.0] - 2023-5-11
+### :magic_wand: Added
+- `AuroraHostListProvider` to check for stale writer records obtained after writer-failover so that they are not used ([PR #435](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/435)).
```

Comment:
might rephrase slightly, it sounds a bit like AuroraHostListProvider is a new class we added with the way its currently phrased.

`AuroraHostListProvider` checks for stale writer records after writer failover so that they are not used ([PR #435](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/435))